src/CoreWCF.Primitives/src/CoreWCF/Channels/ByteStreamMessageEncodingBindingElement.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.ComponentModel;
+using System.Xml;
+using CoreWCF.Runtime;
+
+namespace CoreWCF.Channels
+{
+    public sealed class ByteStreamMessageEncodingBindingElement : MessageEncodingBindingElement
+    {
+        private readonly XmlDictionaryReaderQuotas _readerQuotas;
+
+        public ByteStreamMessageEncodingBindingElement() : this((XmlDictionaryReaderQuotas)null)
+        {
+        }
+
+        public ByteStreamMessageEncodingBindingElement(XmlDictionaryReaderQuotas quota)
+        {
+            _readerQuotas = new XmlDictionaryReaderQuotas();
+            if (quota != null)
+            {
+                quota.CopyTo(_readerQuotas);
+            }
+        }
+
+        ByteStreamMessageEncodingBindingElement(ByteStreamMessageEncodingBindingElement byteStreamEncoderBindingElement)
+            : this(byteStreamEncoderBindingElement._readerQuotas)
+        {
+        }
+
+        public override MessageVersion MessageVersion
+        {
+            get
+            {
+                return MessageVersion.None;
+            }
+            set
+            {
+                if (value != MessageVersion.None)
+                {
+                    throw Fx.Exception.Argument("MessageVersion", SR.Format(SR.ByteStreamMessageEncoderMessageVersionNotSupported, value));
+                }
+            }
+        }
+
+        public XmlDictionaryReaderQuotas ReaderQuotas
+        {
+            get
+            {
+                return _readerQuotas;
+            }
+            set
+            {
+                if (value == null)
+                    throw Fx.Exception.ArgumentNull("ReaderQuotas");
+                value.CopyTo(ReaderQuotas);
+            }
+        }
+
+        // TODO: Channel factories?
+        //public override bool CanBuildChannelFactory<TChannel>(BindingContext context)
```

Comment:
Remove commented out code

Reply:
This is addressed in 4c1b39d3c168dd167891ea5b0d7bece926b6f277.